### PR TITLE
phone login problem fix

### PR DESCRIPTION
**Pull request by a4adityaa**

### CODE REVIEW

File: lib/src/app/feature/authentication/presentation/viewmodel/auth_view_model.dart [hunk 1]
```diff
@@ -99,13 +99,18 @@ class AuthViewModel extends ChangeNotifier {
...
 
   Future<bool> phoneSignIn() async {
+    verified = true;
+
     firebaseAuthInstance = FirebaseAuth.instance;
     final UserCredential userCredential =
         await firebaseAuthInstance.signInWithCredential(phoneAuthCredential!);
 
     if ((userCredential.user != null)) {
+      verified = true;
```
Comment (Abhishek01039) [inline lib/src/app/feature/authentication/presentation/viewmodel/auth_view_model.dart:109]: No need to create a variable. just return true and false

File: lib/src/app/feature/authentication/presentation/viewmodel/auth_view_model.dart [hunk 2]
```diff
@@ -77,7 +77,7 @@ class AuthViewModel extends ChangeNotifier {
   Future<bool> verifySmsCodeWeb() async {
     try {
       final UserCredential? userCredential =
-          await confirmationResult?.confirm(smsCode);
+          await confirmationResult?.confirm('$smsCode');
```
Comment (Abhishek01039) [inline lib/src/app/feature/authentication/presentation/viewmodel/auth_view_model.dart:80]: Please revert this changes, it's already same as pervious version

**Review by Abhishek01039 — CHANGES_REQUESTED** [review]

**Comment by a4adityaa** [comment]
https://user-images.githubusercontent.com/114215653/194797682-1a2ade2e-4b56-4d55-b747-b45ff684f5aa.mp4

**Review by Abhishek01039 — APPROVED** [review]
LGTM!

**Pull request merged**